src/compressed_tensors/transform/transform_config.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict
+
+from compressed_tensors.transform import TransformArgs, TransformScheme
+from pydantic import BaseModel
+
+
+__all__ = ["TransformsConfig"]
+
+
+class TransformsConfig(BaseModel):
```

Comment:
This should either be `TransformConfig` or the file should be `transforms_config.py`. I understand `TransformsConfig` as the highest scope, and containing multiple `TransformArgs` and `TransformScheme`s, but I think `TransformConfig` is a better name for consistency